cmd/containerd-shim-runhcs-v1/pod.go

```diff
@@ -152,21 +152,8 @@ func createPod(ctx context.Context, events publisher, req *task.CreateTaskReques
 		// isolated. Process isolated WCOW gets the namespace endpoints
 		// automatically.
 		if parent != nil {
-			nsid := ""
 			if s.Windows != nil && s.Windows.Network != nil {
-				nsid = s.Windows.Network.NetworkNamespace
-			}
-
-			if nsid != "" {
```

Comment:
Should we not keep the same check of if the namespace != "" before calling the new SetupNetworkNamespace?

Reply:
Good catch. I kind of assumed that NetworkNamespace won't be empty if s.Windows.Network is non nil.